alpha/apps/kaltura/lib/batch2/model/kConvertJobData.php

```diff
@@ -245,6 +246,15 @@ function calculateUrgency(BatchJob $batchJob) {
 		if($allNoImpact)
 			$readiness = flavorParamsConversionProfile::READY_BEHAVIOR_REQUIRED;
 
+		// Case 4
+		$jobEntry = $batchJob->getEntry();
+		if (!empty($jobEntry))
```

Comment:
Instead of if (!empty($jobEntry)) use if($entry), It does the same , and more commonly used, plus less code.